sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/aad/AADAuthenticationProperties.java

```diff
@@ -274,7 +278,15 @@ public String getBaseUri() {
     }
 
     public void setBaseUri(String baseUri) {
-        this.baseUri = baseUri;
+        this.baseUri = addSlash(baseUri);
+    }
+
+    public String getGraphBaseUri() {
+        return graphBaseUri;
+    }
+
+    public void setGraphBaseUri(String graphBaseUri) {
+        this.graphBaseUri = addSlash(graphBaseUri);
```

Comment:
Move `addSlash` into `afterPropertiesSet`. Both `baseUri` and `graphBaseUri` need `addSlash`